infrastructure/docker/build/Dockerfile-cache-config

```diff
@@ -37,9 +37,14 @@ RUN if [[ ${RHEL_VERSION%%.*} -ge 8 ]]; then \
 		git \
 		rpm-build \
 		rsync \
+		gzip \
 		epel-release && \
 	yum -y clean all
 
+RUN yum -y install dnf-plugins-core
+RUN yum -y config-manager --set-enabled PowerTools || yum -y config-manager --set-enabled powertools
```

Comment:
This layer fails to build when building for CentOS 7:

```
```

```shell
[user@computer trafficcontrol]$ RHEL_VERSION=7 docker-compose -f infrastructure/docker/build/docker-compose.yml build cache-config_build
[...]
#7 1.814 Command line error: no such option: --set-enabled
#7 ERROR: executor failed running [/bin/sh -c yum -y config-manager --set-enabled PowerTools || yum -y config-manager --set-enabled powertools]: exit code: 1
------
 > [common-dependencies 4/5] RUN yum -y config-manager --set-enabled PowerTools || yum -y config-manager --set-enabled powertools:
------
executor failed running [/bin/sh -c yum -y config-manager --set-enabled PowerTools || yum -y config-manager --set-enabled powertools]: exit code: 1
Service 'cache-config_build' failed to build : Build failed
```

Reply:
In CDN in a Box, we handle this with an `if` statement and an `enable_repo` variable:
https://github.com/apache/trafficcontrol/blob/c90a2bc3a415240d916479d6e0c1447629b5ac7f/infrastructure/cdn-in-a-box/traffic_ops/Dockerfile#L35-L45

Reply:
Fixed.